sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/AspNetHostedService/tests/Azure.Messaging.EventHubs.Processor.HostedService.Samples.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net462</TargetFrameworks>
```

Comment:
Should we consider only including `net462` when building from the Azure SDK repo?  It really exists just to service our common text matrix.

Reply:
@jsquire Agreed. I've added this to the props file.